backend/app/services/api/filterer.rb

```diff
@@ -62,6 +65,12 @@ def filter_by_only_verified
       self.query = query.where trusted: true
     end
 
+    def filter_by_relations
```

Comment:
Hmm, at least for `priority_landscape` that would not be necessary to make joins and complicate this, it could be done like filtering by attribute, `Project.where(priority_landscape: id)`

Reply:
Looking at specs I see this is used for to filter both project developers and projects, ok then discard my latest comment.

Reply:
But we have join and without distinct, would that not return duplicates when filtering by multiple priority landscapes?

Reply:
Yeah, you are right with this one, I have decided to keep code simple so it works same way for has_many (PD has many priority landscapes) and belongs_to (Project has one priority landscape) association. To be honest, I don't think it will slow down query much, but if you want, I can refactor this method so belongs to associations are handled differently.

Reply:
Nope, duplicates will be removed thanks to `pluck` (`original_query.where id: query.pluck(:id)`). My initial intention for adding pluck there was to split filter queries and rest of the stuff, but it also solves issue with duplicates

Reply:
Ok, I see that later from this query `id` is plucked. Then all good

Reply:
Anyway, tested with api docs.

Reply:
Lol, I did not see your comments. Seeing them just now :D haha